src/components/navBar/navBar.js

```diff
@@ -414,6 +418,7 @@ function MdNavItem($$rAF) {
       'mdNavHref': '@?',
       'mdNavSref': '@?',
       'name': '@',
+      'disabled': '=?ngDisabled',
```

Comment:
Yeah that's why `disabled` doesn't work on the tab item as well.

- http://codepen.io/DevVersion/pen/rrrBzL

Watching for the `disabled` attribute ensures that `ngDisabled` and the normal `disabled` attribute work.